net_util/src/open_tap.rs

```diff
@@ -85,10 +85,23 @@ pub fn open_tap(
     for i in 0..num_rx_q {
         let tap: Tap;
         if i == 0 {
-            tap = match if_name {
-                Some(name) => Tap::open_named(name, num_rx_q, flags).map_err(Error::TapOpen)?,
-                None => Tap::new(num_rx_q).map_err(Error::TapOpen)?,
+            let created: bool;
+            (tap, created) = match if_name {
+                Some(name) => (
+                    Tap::open_named(name, num_rx_q, flags).map_err(Error::TapOpen)?,
+                    false,
+                ),
+                None => (Tap::new(num_rx_q).map_err(Error::TapOpen)?, true),
             };
+
+            // If a Tap device was not only opened but created in the system by Cloud Hypervisor
+            if created {
+                log::info!(
+                    "Created tap device: name={}, num_rx_q={num_rx_q}",
+                    String::from_utf8_lossy(tap.get_if_name())
+                );
+            }
```

Comment:
Wouldn't it be simpler to just log inline when the tap is created? Then there's no need for this bool at all.